devfile.yaml

```diff
@@ -17,12 +26,34 @@ components:
     args:
       - '-c'
       - >-
-        while [ ! -f /projects/kubernetes-api/.theia/settings.json -o 
-        ! -f /projects/.theia/settings.json ]; do echo "Waiting for the clone end" ; sleep 2; done;
-        sleep 2 ; echo "Updating the yaml schema bindings" ; sed -e
-        's:/schemas/:/kubernetes-api/schemas/:g'
-        /projects/kubernetes-api/.theia/settings.json >
-        /projects/.theia/settings.json; sleep infinity
+        while [ ! -f
+        ${CHE_PROJECTS_ROOT}/src/github.com/devfile/kubernetes-api/.theia/settings.json
+        -o  ! -f ${CHE_PROJECTS_ROOT}/.theia/settings.json ]; do echo "Waiting
+        for the clone end" ; sleep 2; done; sleep 2 ; echo "Updating the yaml
+        schema bindings" ; sed -e
+        's:/schemas/:/src/github.com/devfile/kubernetes-api/schemas/:g'
+        ${CHE_PROJECTS_ROOT}/src/github.com/devfile/kubernetes-api/.theia/settings.json
```

Comment:
It's like init containers, right? Can we somehow avoid it?

Reply:
Well, I can't avoid that for now, since the devfile 1.0 plugin preferences do not support structured content.
And I have to be sure that the sources were cloned by Che-theia, before completing the global Theia settings with the yaml-schema ones provided in the GH project.